DOCS/man/options.rst

```diff
@@ -2607,11 +2607,11 @@ Subtitles
     is removed).
 
 ``--sub-forced-only=<auto|yes|no>``
-    Display only forced subtitles for the DVD subtitle stream selected by e.g.
-    ``--slang`` (default: ``auto``). When set to ``auto``, enabled when the
-    ``--subs-with-matching-audio`` option is on and a non-forced stream is selected.
-    Enabling this will hide all subtitles in streams that don't make a distinction
-    between forced and unforced events within a stream.
+    Display only forced subtitles for the DVD subtitle stream selected by
+    e.g. ``--slang`` (default: ``auto``). When set to ``auto``, and the
+    ``--subs-with-matching-audio`` option is set, a non-forced stream will
+    be selected. Enabling this will hide all subtitles in streams that don't
```

Comment:
I don't think this is correct. This doesn't influence stream selection, only display, right?

Reply:
I guess that was also a poor choice of words. A better way putting it is:

> a non-forced stream will be **displayed**.